Description: Deselects the "Notify all members" checkbox by default
Credits: novie

diff --git a/Telegram/SourceFiles/boxes/pin_messages_box.cpp b/Telegram/SourceFiles/boxes/pin_messages_box.cpp
index 3b9718ceb..45a1a3c60 100644
--- a/Telegram/SourceFiles/boxes/pin_messages_box.cpp
+++ b/Telegram/SourceFiles/boxes/pin_messages_box.cpp
@@ -72,7 +72,7 @@ void PinMessageBox::prepare() {
 		_notify.create(
 			this,
 			tr::lng_pinned_notify(tr::now),
-			true,
+			false,
 			st::defaultBoxCheckbox);
 		_checkbox = _notify;
 	}
